specs/utility/osquery_info.table

```diff
@@ -11,7 +11,8 @@ schema([
     Column("build_platform", TEXT, "osquery toolkit build platform"),
     Column("build_distro", TEXT, "osquery toolkit platform distribution name (os version)"),
     Column("start_time", INTEGER, "UNIX time in seconds when the process started"),
-    Column("watcher", INTEGER, "Process (or thread/handle) ID of optional watcher process")
+    Column("watcher", INTEGER, "Process (or thread/handle) ID of optional watcher process"),
+    Column("platform_mask", INTEGER, "The osquery platform bitmask"),
```

Comment:
I like this idea, but aside from searching the code base it might not be entirely obvious what the numeric values map to. Is there somewhere we can document the potential values of this bitmask and then link it in this `Column` description?

Reply:
That's a good question.

I tend to prefer self documenting, but that becomes way too verbose.

I'm leery of making docs, we already have a lot, I think they'll skew. What it we linked to the source code from here?